app/pages/search/search-controls/SearchControls.spec.js

```diff
@@ -50,20 +50,20 @@ describe('pages/search/search-controls/SearchControls', () => {
     });
 
     describe('isFavorite control', () => {
-      function getIsFavoriteControlWrapper(values) {
+      function getIsFavoriteCheckboxWrapper(values) {
         const wrapper = getWrapper({
           initialValues: Object.assign({}, defaultInitialValues, values),
         });
-        return wrapper.find('[controlId="is-favorite-control-group"]');
+        return wrapper.find('.is-favorite-checkbox');
       }
 
       it('has correct label', () => {
-        const controlLabel = getIsFavoriteControlWrapper().find(ControlLabel);
+        const controlLabel = getIsFavoriteCheckboxWrapper();
```

Comment:
- [ ] this could be 
```javascript
const label = getIsFavoriteCheckboxWrapper().prop('children');
expect(label).to.equal('Näytä vain suosikit');
```